.circleci/config.yml

```diff
@@ -201,11 +201,8 @@ workflows:
       - default_test_job:
           requires:
             - build
-          prefixTestTask: true
-          name: test_<< matrix.testTask >>
-          matrix:
-            parameters:
-              testTask: ["8"]
```

Comment:
Do you mind keeping the parameterization here? Really hoping to bring back testing on Java 11 and Java 14 soon

Reply:
I hope to switch to GitHub Actions sooner :)